doc/installing.rst

```diff
@@ -0,0 +1,49 @@
+.. _installing:
+
+Installing pyRiemann
```

Comment:
Same remark, this file must be updated, to describe pyRiemann-qiskit instead of pyRiemann.